tools/load_demo_data.py

```diff
@@ -3,15 +3,14 @@
 import subprocess
 import base64
 from pathlib import Path
-import shutil
 import pandas as pd
 import signal
 import requests
 
 from baselayer.app.env import load_env
 from baselayer.app.model_util import status, create_tables, drop_tables
 from social_tornado.models import TornadoStorage
-from skyportal.models import init_db, Base, DBSession, Source, User
+from skyportal.models import init_db, Base, Candidate, DBSession, Source, User
```

Comment:
add both saved and unsaved candidates during load_demo_data with associated photometry, spectra, thumbnails, etc...